Automattic/sensei
# Add stylesheets for course theme variations

Resolves the issue of having to write css styles in variation json files.

## Proposed Changes
* Based on the new ability of Course theme of adding body class for variation and firing hook for body class, we're adding stylesheets in sensei that gets loaded based on the selected variation.

## Testing Instructions


1. Make sure you are on latest course theme trunk
2. If you are running "npm run start", stop and restart it
3. Select a variation of course theme
4. Add some random style in the stylesheet applicable for that variation for testing purposes (like `* { color: red !important; }`)
5. Check if the style gets applied on the frontend as expected.

## Pre-Merge Checklist
